packages/@aws-cdk/aws-codepipeline/lib/pipeline.ts

```diff
@@ -260,9 +258,17 @@ export class Pipeline extends PipelineBase {
     this.artifactBucket.grantReadWrite(this.role);
     this.pipelineName = this.getResourceNameAttribute(codePipeline.ref);
     this.pipelineVersion = codePipeline.attrVersion;
-    this.crossRegionReplicationBuckets = props.crossRegionReplicationBuckets || {};
     this.crossRegionBucketsPassed = !!props.crossRegionReplicationBuckets;
-    this.artifactStores = {};
+
+    for (const region in props.crossRegionReplicationBuckets) {
```

Comment:
Use `for (const [k,v] of Object.entries())` instead of `for (x ...in)`

Reply:
Done.